scripts/rebase-fork.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
```

Comment:
Suggest to change to using the env for bash in case folks don't have bash installed in `/bin`:
```suggestion
#!/usr/bin/env bash
```